mycore-viewer/src/main/typescript/modules/pdf/components/MyCoRePDFViewerComponent.ts

```diff
@@ -29,9 +29,10 @@ namespace mycore.viewer.components {
             var that = this;
 
             // see MV-53
-            (<any>PDFJS).disableAutoFetch=true;
-            (<any> PDFJS).cMapUrl=this._settings.webApplicationBaseURL + "/modules/iview2/cmaps/";
-            (<any> PDFJS).cMapPacked = true;
+            (<any>pdfjsLib).disableAutoFetch=true;
+            (<any>pdfjsLib).cMapUrl=this._settings.webApplicationBaseURL + "/modules/iview2/cmaps/";
+            (<any>pdfjsLib).cMapPacked = true;
+            (<any>pdfjsLib).disableOpenActionDestination = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['pdfjsLib' is not defined.](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5286493)